framework/src/main/java/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -391,6 +424,41 @@ public void setRoot(@Nullable CompilationUnitTree root) {
             this.flowResultAnalysisCaches.clear();
             this.initializerCache.clear();
             this.defaultQualifierForUseTypeAnnotator.clearCache();
+
+            if (this.checker.getParentChecker() == null) {
+                // This is an ultimate parent checker, so after it runs the shared CFG it is using
+                // will be dead.
```

Comment:
What does it mean for the CFG to be dead?  You mean null?

Reply:
I meant that it's no longer in use (like a variable that is dead in a liveness analysis). I'll update this comment to make it more clear.